project/Build.scala

```diff
@@ -1069,6 +1069,49 @@ object Build extends sbt.Build {
   // Testing
 
   val testTagSettings = Seq(
+      javaOptions in Test ++= {
```

Comment:
There would be only:

```scala
val sems = (scalaJSStage in Test).value match {
  case FastOptStage => (scalaJSSemantics in (Test, fastOptJS)).value
  case FullOptStage => (scalaJSSemantics in (Test, fullOptJS)).value
}
```
And it would change all line indentations there. Do you really want me to do itt?